samples/ExampleGallery/UAP/CameraEffectExample.xaml.cs

```diff
@@ -119,6 +122,12 @@ private void EffectCombo_SelectionChanged(object sender, SelectionChangedEventAr
             var task = SetEffect(effect);
         }
 
+        private void BlurAmountSlider_ValueChanged(object sender, RangeBaseValueChangedEventArgs e)
+        {
+            if (effectPropertySet == null) return;
+            effectPropertySet["BlurAmount"] = e.NewValue;
```

Comment:
Slider's ValueChanged event updates the property bag's BlurAmount property. This value is used by the DynamicBlurVideoEffect when applying Win2D Gaussian Blur when processing the video frame.